### ISSUE TITLE
Ship defaults in /etc/xdg/pqivrc instead?

### ISSUE THREAD

**Issue by tleydxdy** [phillipberndt/pqiv#246]
Hi, I wonder if it's preferable to ship the defaults in /etc/xdg/pqivrc instead of hard-coding it in the source (perhaps hardcoding the most basic commands like quit() makes sense).

What prompted this for me is that the default binds a lot of keys and especially lots of keys to the same function. Personally I get really confused when I accidentally hit some key and it does something surprising to me (e.g. I unbond all the keys in mpv that I don't use). But doing so in pqiv is quite tedious and would also break if a new default bind is added.

I think that changing this would also have a few side benefits. It would serves as a reference on how the config format works, and it also makes the defaults self-documenting on the user's machine, instead of requiring the man page to be synced or dedicated commands that prints out these information.

I understand that this is quite a big change, and I'm willing to work on it. I just want to see if this is desirable or not.

**Comment by phillipberndt** [comment]
How about a "drop all existing key bindings" instruction to put at the top of your config? That sounds like an unobstrusive, simple change. And IIUC it would give you what you're looking for?!

**Comment by tleydxdy** [comment]
Yes, that would solve the primary issue. Actually, it would probably also enable shipping a default config while being backwards compatible.
